packages/snaps-controllers/src/cronjob/CronjobController.ts

```diff
@@ -157,22 +170,26 @@ export class CronjobController extends BaseController<
 > {
   readonly #timers: Map<string, Timer>;
 
+  readonly #stateManager: CronjobControllerStateManager;
+
   #dailyTimer: Timer = new Timer(DAILY_TIMEOUT);
 
-  constructor({ messenger, state }: CronjobControllerArgs) {
+  constructor({ messenger, state, stateManager }: CronjobControllerArgs) {
     super({
       messenger,
       metadata: {
-        events: { persist: true, anonymous: false },
+        events: { persist: false, anonymous: false },
       },
       name: controllerName,
       state: {
         events: {},
         ...state,
+        ...stateManager.getInitialState(),
```

Comment:
I guess this means we will always overwrite an obsolete state after the first init but that looks fine for now.

Reply:
Yeah and we still need to recover the state the first time